### DISCUSSION TITLE
Updating from beta to beta then into prod?

### DISCUSSION THREAD

**Discussion by nealhamiltonjr** [mealie-recipes/mealie#1344] [General]
Is it safe to assume that the data in beta 2 will be compatible with beta 3 and etc ..then into the final release? I'd like to start and populating the beta 2 but don't want to waste time if it all has to be redone as we move from beta to beta then into the final release. I'll still test and report bugs but I don't want to get excited and start putting in a lot of recipes only to have to redo it.

**Comment by hay-kot** [comment]
The beta's are considered "Stable" in that there shouldn't be any data loss from beta to beta to eventually stable release. We've already done several database migrations in the beta process, so you should be safe to upgrade. As always read the change log before you upgrade though.

**Reply by nealhamiltonjr** [reply]
Thanks, and thanks for the time and energy you're putting into this awesome piece of software.
